lib/gds_api/support_api.rb

```diff
@@ -47,4 +47,13 @@ def organisation(organisation_slug)
   def feedback_export_request(id)
     get_json!("#{endpoint}/anonymous-feedback/export-requests/#{id}")
   end
+
+  def problem_reports(options = {})
```

Comment:
Sure thing @Davidslv  - please see updated comments in `lib/gds_api/support_api.rb`